resources/puppetlabs/lein-ezbake/template/global/ext/redhat/ezbake.service.erb

```diff
@@ -49,9 +49,15 @@ ExecStartPre=<%= action %>
 <% end -%>
 <% end -%>
 
-ExecReload=/opt/puppetlabs/server/apps/<%= EZBake::Config[:real_name] %>/bin/<%= EZBake::Config[:real_name] %> reload
-ExecStart=/opt/puppetlabs/server/apps/<%= EZBake::Config[:real_name] %>/bin/<%= EZBake::Config[:real_name] %> start
-ExecStop=/opt/puppetlabs/server/apps/<%= EZBake::Config[:real_name] %>/bin/<%= EZBake::Config[:real_name] %> stop
+ExecStart="$JAVA_BIN" $JAVA_ARGS $LOG_APPENDER \
+  '-XX:OnOutOfMemoryError=kill -9 %p' -XX:+CrashOnOutOfMemoryError \
+  -XX:ErrorFile="$LOGS_DIRECTORY/<%= EZBake::Config[:real_name] %>_err_pid%p.log" \
+  -cp "$INSTALL_DIR/<%= EZBake::Config[:uberjar_name] %>" \
+  clojure.main \
+  -m <%= EZBake::Config[:main_namespace] %> \
+  --config "$CONFIG" \
+  --bootstrap-config "$BOOTSTRAP_CONFIG" \
+  $TK_ARGS
```

Comment:
I also considered some script in `libexec` for this. @nmburgan I'd appreciate your thoughts on this design wise.

Reply:
Personally I prefer to keep it in the unit. Makes it easier to debug.

Reply:
Yeah, I think so too.  I don't know the full impact this change would have (I've worked more with the Ruby services than the Clojure ones), but I think each app having a reload/start/stop is nice in case we need to make app-specific changes to the command.